package com.kroger.ngpp.testutils.exceptions;

public class TestSetupException extends RuntimeException {
    public TestSetupException(String message, Throwable cause) {
        super(message, cause);
    }
}
